virttest/shared/cfg/guest-hw.cfg

```diff
@@ -24,7 +24,9 @@ variants:
         required_qemu = [2.6.0, )
         nic_model = e1000e
     - virtio_net:
-        nic_model = virtio
+        nic_model = virtio-net-pci
```

Comment:
You should consider different naming in different platforms.
https://github.com/avocado-framework/avocado-vt/blob/11a2c5a3eded430db30740b642d02f3d8a1279eb/virttest/qemu_vm.py#L606-L613

Reply:
Thanks for your reminder. After you reminded me yesterday, I forgot it when I updated it. Thank you so much.